### PR TITLE
CI is enforced over java code to demand basic code quality

### PR DESCRIPTION
**Pull request by relentless-pursuit**
Part of #107

### CODE REVIEW

File: EmptyBlock/Example3/config.xml [hunk 1]
```diff
@@ -23,7 +23,7 @@
         <property name="javaParseExceptionSeverity" value="ignore"/>
 
         <module name="EmptyBlock">
-            <property name="tokens" value="LITERAL_DEFAULT"/>
+            <property name="tokens" value="LITERAL_CASE,LITERAL_DEFAULT"/>
```
Comment (romani) [inline EmptyBlock/Example3/config.xml:26]: https://checkstyle.org/checks/blocks/emptyblock.html#Example3-config

It is not what we have.
https://github.com/checkstyle/checkstyle/tree/master/src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/blocks/emptyblock there are only 2 examples 

3rd at https://github.com/checkstyle/checkstyle/blob/master/src/xdocs-examples/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/blocks/emptyblock/Example3.java
It was recently uploaded

File: PatternVariableName/all-examples-in-one/config.xml [hunk 2]
```diff
@@ -36,6 +36,30 @@
             <property name="id" value="example3"/>
         </module>
 
+        <module name="PatternVariableName">
+            <property name="format" value="^[A-Z][A-Z0-9]*$"/>
+            <property name="id" value="example4"/>
+            <property name="id" value="example4"/>
```
Comment (romani) [inline PatternVariableName/all-examples-in-one/config.xml:42]: Why twice id ?

**Review by romani — APPROVED** [review]
Items to improve your in next PRs

File: EmptyBlock/Example3/config.xml (hunk 1 above)
Comment (romani) [reply EmptyBlock/Example3/config.xml:26]: All good here, no action is required.

File: PatternVariableName/all-examples-in-one/config.xml (hunk 2 above)
Comment (romani) [reply PatternVariableName/all-examples-in-one/config.xml:42]: This should be fixed

Comment (relentless-pursuit) [reply PatternVariableName/all-examples-in-one/config.xml:42]: > This should be fixed

But, i guess the example is written that way. 
https://github.com/checkstyle/checkstyle/blob/f1a36722c5038aa104f7be12486cf2d7edff37ee/src/xdocs-examples/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/naming/patternvariablename/Example4.java

Comment (relentless-pursuit) [reply PatternVariableName/all-examples-in-one/config.xml:42]:
```
    <module name="PatternVariableName">
        <property name="id" value="FinalPatternVariableName"/>
        <property name="format" value="^[A-Z][A-Z0-9]*$"/>
    </module>

    <module name="PatternVariableName">
        <property name="id" value="NonFinalPatternVariableName"/>
        <property name="format" value="^([a-z][a-zA-Z0-9]*|_)$"/>
    </module>

    <module name="SuppressionXpathSingleFilter">
      <property name="id" value="FinalPatternVariableName"/>
      <property name="query" value="//PATTERN_VARIABLE_DEF[
            not(./MODIFIERS/FINAL)]/IDENT"/>
    </module>

    <module name="SuppressionXpathSingleFilter">
      <property name="id" value="NonFinalPatternVariableName"/>
      <property name="query" value="//PATTERN_VARIABLE_DEF[
            (./MODIFIERS/FINAL)]/IDENT"/>
```

Comment (relentless-pursuit) [reply PatternVariableName/all-examples-in-one/config.xml:42]: All these are multiple modules under example 4. And, therefore we in all in one, we attach each module along with which example it was traced from.

Comment (romani) [reply PatternVariableName/all-examples-in-one/config.xml:42]: Please create issue to update extractor to keep id same as in example if it was defined.
No duplicates of IDs.

Comment (romani) [reply PatternVariableName/all-examples-in-one/config.xml:42]: https://github.com/checkstyle/test-configs/issues/145

**Pull request merged**